releasenotes/notes/use-remote-tagger-for-trace-and-security-agents-9650d1be5d49e47f.yaml

```diff
@@ -0,0 +1,11 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+deprecations:
+  - |
+    ``apm_config.remote_tagger`` and ``security_agent.remote_tagger`` are deprecated and removed.
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏The configurations are not documented (not part of the config template) so it seems odd to make a release note to announce that they are removed

Reply:
I found a reference on the `CHANGELOG.rst` https://github.com/DataDog/datadog-agent/blob/a8b89968b48e3b5288d0ae5e6796447445dcec45/CHANGELOG.rst#L6199

Reply:
can you also mention the system-probe flag here ?